pkgs/development/lua-modules/generated-packages.nix

```diff
@@ -184,7 +184,7 @@ buildLuarocksPackage {
   pname = "cassowary";
   version = "2.3.2-1";
   knownRockspec = (fetchurl {
-    url    = "mirror://luarocks/cassowary-2.3.2-1.rockspec";
+    url    = "https://raw.githubusercontent.com/rocks-moonscript-org/moonrocks-mirror/master/cassowary-2.3.2-1.rockspec";
```

Comment:
damn this updater that keeps chaning the urls, if you wanna fix this, please go ahead, meanwhile can we avoid the change ? with `git add -p` for instance